cargo-concordium/src/main.rs

```diff
@@ -552,6 +613,31 @@ pub fn main() -> anyhow::Result<()> {
                     .context("Could not write base64 schema file.")?;
             }
         }
+        Command::SchemaTemplate {
+            out,
+            module_path,
+            schema_path,
+            wasm_version,
+        } => {
+            let schema = get_schema(module_path, schema_path, wasm_version)
+                .context("Could not get schema.")?;
+
+            if out.as_path() == Path::new("-") {
+                write_schema_template(None, &schema)
+                    .context("Could not print the template of the schema.")?;
+            } else {
+                // A valid path needs to be provided when using the `--out` flag.
+                if out.file_name().is_none() || out.is_dir() {
+                    anyhow::bail!(
+                        "The `--out` flag should point to an existing directory + filename \
```

Comment:
I'm not sure the existing directory part is true. You create the parent path in `write_schema_tempalte`.